crates/pop-parachains/templates/pallet/advanced_mode/src/mock.rs.templ

```diff
@@ -0,0 +1,174 @@
+{{- let using_runtime_event = pallet_common_types|contains(TemplatePalletConfigCommonTypes::RuntimeEvent) -}}
+{{- let using_runtime_origin = pallet_common_types|contains(TemplatePalletConfigCommonTypes::RuntimeOrigin) -}}
+{{- let using_currency = pallet_common_types|contains(TemplatePalletConfigCommonTypes::Currency) -}}
+use crate::{self as pallet_^^name|lower^^, *};
+use pop_dev::{PlaceholderType, pop_todo};
+
+use frame::{
+	deps::frame_support::{
```

Comment:
See `Usage section here https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_frame/index.html

it is all about `preludes` :) 

and whatever is not in prelude, but reasonable should be: 

1. temporarily fetch it via `dpes`
2. either PR or, make an issue for us to fix. 

In general we should try and avoid using `deps`, if that makes sense.

Reply:
```rust 
use frame::{prelude::*, runtime::prelude::*, testing_prelude::*, deps::frame_support::runtime};
```

compiles with `cargo test`, but it doesn't without the last import. It seems the `runtime` isn't in the preludes

Reply:
This is the PR that you are missing: 

https://github.com/paritytech/polkadot-sdk/pull/5151

Hopefully you will have it backported soon enough :)